.github/workflows/scale-test-egw.yaml

```diff
@@ -0,0 +1,485 @@
+          if [ -n "${SHA}" ]; then
+            echo sha=${SHA} >> $GITHUB_OUTPUT
+            CILIUM_INSTALL_DEFAULTS+=" --helm-set=image.repository=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/cilium-ci \
+            --helm-set=image.useDigest=false \
+            --helm-set=image.tag=${SHA} \
+            --helm-set=operator.image.repository=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/operator \
+            --helm-set=operator.image.suffix=-ci \
+            --helm-set=operator.image.tag=${SHA} \
+            --helm-set=operator.image.useDigest=false \
+            --helm-set=clustermesh.apiserver.image.repository=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/clustermesh-apiserver-ci \
+            --helm-set=clustermesh.apiserver.image.tag=${SHA} \
+            --helm-set=clustermesh.apiserver.image.useDigest=false \
+            --helm-set=hubble.relay.image.repository=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/hubble-relay-ci \
+            --helm-set=hubble.relay.image.tag=${SHA} \
+            --helm-set=hubble.relay.image.useDigest=false"
+          fi
+
+          OWNER="${{ github.ref_name }}"
+          OWNER="${OWNER/./-}"
+
+          if [ "${{ github.event_name }}" == "workflow_dispatch" ] ; then
+            NUM_CLIENT_PODS="${{ inputs.num-clients }}"
+            CLIENT_QPS="${{ inputs.client-qps }}"
+          else
+            NUM_CLIENT_PODS="100"
+            CLIENT_QPS="20"
+          fi
+
+          TEST_NAME="${{ env.test_name }}-${{ matrix.test_type }}-${NUM_CLIENT_PODS}-${CLIENT_QPS}"
+
+          CLUSTER_NAME="${TEST_NAME}-${{ env.cluster_name }}"
+
+          # Nodes holding the EGW clients use a maximum of 100 pods per node.
+          # Add an extra as a buffer.
+          NUM_CLIENT_NODES="$(((NUM_CLIENT_PODS / 100) + 1))"
+
+          echo sha=${SHA} >> $GITHUB_OUTPUT
+          echo cilium_install_defaults=${CILIUM_INSTALL_DEFAULTS} >> $GITHUB_OUTPUT
+          echo owner=${OWNER} >> $GITHUB_OUTPUT
+          echo test_name=${TEST_NAME} >> $GITHUB_OUTPUT
+          echo cluster_name=${CLUSTER_NAME} >> $GITHUB_OUTPUT
+          echo num_client_pods=${NUM_CLIENT_PODS} >> $GITHUB_OUTPUT
+          echo num_client_nodes=${NUM_CLIENT_NODES} >> $GITHUB_OUTPUT
+          echo client_qps=${CLIENT_QPS} >> $GITHUB_OUTPUT
+
+      - name: Wait for images to be available
+        timeout-minutes: 30
+        shell: bash
+        run: |
+          for image in cilium-ci operator-aws-ci hubble-relay-ci ; do
+            until docker manifest inspect quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/$image:${{ steps.vars.outputs.sha }} &> /dev/null; do sleep 45s; done
+          done
+
+      - name: Wait for EGW scale utils image to be available
+        timeout-minutes: 30
+        shell: bash
+        run: |
+          # Run this seprate from the other "Wait for images to be available" step to help with debugging.
+          until docker manifest inspect quay.io/cilium/egw-scale-utils:${{ env.egw_utils_ref }} &> /dev/null; do sleep 45s; done
```

Comment:
There's likely no need to retry, as the image should be always there.

Reply:
I left the step in as a sanity check, but remove the retry behavior. WDYT?